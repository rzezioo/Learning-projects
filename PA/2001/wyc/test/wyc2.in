107 33
-1 -32
-1 -4
-1 -20
-1 28
1 -5
1 28
1 -3
-2 -14
-2 -16
-2 4
-2 -6
-2 -17
-2 -29
-2 2
2 -26
2 3
-3 -10
-3 17
-3 -4
3 -28
3 30
3 26
3 -15
-4 -23
-4 -17
-4 -16
-4 33
-5 20
-5 -23
-5 33
-6 -8
-6 -22
-6 -13
6 29
6 22
6 -26
-7 -21
-7 -18
-7 32
7 -22
7 -10
-8 -18
8 -25
8 16
8 -8
-9 21
-9 -30
9 23
9 18
9 31
9 -22
-10 26
-10 21
10 -29
10 12
-11 -13
11 -16
11 -23
-12 31
-12 -21
-12 14
12 13
12 -15
-13 -14
13 18
13 -20
13 15
-14 -29
-14 18
-14 -26
14 -15
14 -24
14 32
14 -31
-15 -26
-15 30
-15 -29
-16 -32
-16 -33
-16 18
-16 -19
16 -24
16 29
-17 -32
17 25
-18 28
-18 -33
18 -24
18 19
18 -30
-19 26
19 -33
19 -26
20 -26
20 -33
20 31
-21 30
-22 26
23 -30
24 26
24 25
-25 26
25 30
-27 -28
-27 29
28 -31
29 31
