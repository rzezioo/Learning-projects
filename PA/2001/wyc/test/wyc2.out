15
1 3 6 11 12 14 15 16 17 18 23 24 28 30 31
